typos.toml

```diff
@@ -0,0 +1,13 @@
+[default.extend-words]
+substituters = "substituters"
+# trips on some base64 string in the test
+wel = "wel"
```

Comment:
I doubt this is needed. I think some fixes were added to typos to avoid problems with base64, base32, etc. strings anyway.

Reply:
It triggers on a bunch of addresses as well as hex strings. just a matter of whitelisting all of them eh.

Reply:
https://github.com/crate-ci/typos/blob/6559554303c496ec97efb8673a55f144cef9b983/docs/comparison.md?plain=1#L13 I thought this has been largely solved.

Reply:
You can see from the CI output that it's finding hundreds of "typos" in our gpg key. The reason, presumably, is that a large fraction of lines in an armored gpg key are not valid base64 by themselves.

It also doesn't like `ser` as the name of a module even though this appears in `serde` and probably half the crates in the Rust ecosystem.

It also finds a bunch of actual typos so I'm hesitant to just throw it away ... but yeah, it needs a lot of babying to get this working.

Reply:
Could just ignore `bitcoin/tests/data/`, `SECURITY.md`, add `ser=ser` and other than this seem mostly fine with bunch of legitimate typos.